src/packages.lisp

```diff
@@ -298,24 +352,29 @@ local src-path and sys-path."
                    (set-trans-dep d (dependency-name dep)))
                (progn
                  (format t "~%.... Checking manifest for transitive dependency: ~S" d)
-                 (let ((manifest-package (manifest-get-by-name d)))
-                   (cond ((not manifest-package)
+                 (let ((trans-dependency
+                        (if (manifest-get-by-name d)
+                            (make-trans-dep-from-manifest (manifest-get-by-name d)
+                                                          (dependency-name dep))
+                          (car (member-if
+                                (lambda (x) (string= d (dependency-name x)))
+                                *yaml-packages*)))))
```

Comment:
@CodyReichert would it make sense to just turn everything into a regular dependency?  Is something being a transitive dependency important outside the `installed-dependency-report`?